rules/redis.mk

```diff
@@ -6,7 +6,11 @@ REDIS_TOOLS = redis-tools_$(REDIS_VERSION)_$(CONFIGURED_ARCH).deb
 $(REDIS_TOOLS)_SRC_PATH = $(SRC_PATH)/redis
 $(REDIS_TOOLS)_DEPENDS += $(LIBHIREDIS_DEV)
 $(REDIS_TOOLS)_RDEPENDS += $(LIBHIREDIS)
+# TODO: docker-sonic-p4 depends on redis-tools in Jessie.
+#       Remove this file and src/redis after that resolved.
+ifneq ($(BLDENV),buster)
```

Comment:
What about Stretch? The comment says "docker-sonic-p4 depends on redis-tools in Jessie", but the conditional only checks for Buster. Either update the comment or the conditional.

Reply:
This TODO is not a comment for below block. Since p4 build is broken, I prefer not to touch anything related and build a REDIS_TOOLS in stretch make its fixing in smallest effort in future.

After it is fixed and migrated to stretch or later distro, I will check this TODO and adapt p4 dockerfile to use vanilla redis, and remove this file as TODO mentioned.

---
In reply to: [446458247](https://github.com/Azure/sonic-buildimage/pull/4732#discussion_r446458247) [](ancestors = 446458247)

Reply:
I see. Then consider moving comment to the top of the file.